lms/envs/common.py

```diff
@@ -2235,6 +2235,8 @@
     'city': 'hidden',
     'country': 'hidden',
 }
+REGISTRATION_FOOTER_HTML = ''
```

Comment:
We should give this a more descriptive name, especially since we have an unrelated change in the same PR that actually does show/hide the global footer on the registration page.

Maybe something like `REGISTRATION_FORM_DISCLAIMER_HTML`?
I'm open to suggestions, but we should opt for something more specific than `REGISTRATION_FOOTER_HTML`.